src/components/ReactSlider/ReactSlider.jsx

```diff
@@ -262,6 +267,17 @@ class ReactSlider extends React.Component {
          */
         // eslint-disable-next-line zillow/react/require-default-props
         renderThumb: PropTypes.func,
+
+        /**
+         * Provide a custom render function for the mark node.
+         * The render function will be passed one argument,
+         * an object with props that should be added to your handle node
+         *
+         *     (props) => <span {...props} />
+         *
+         * - `props` {`object`} props to be spread into your track node
+         */
+        renderMark: PropTypes.func,
```

Comment:
It would be great to add a `markClassName`  as well (like we have for `trackClassName` and `thumbClassName`), so that you don't have to specify a render function just to apply a className (which seems to be the most common use-case).